### PR TITLE
Qa/test setup

### PR DESCRIPTION
**Pull request by bhtowles**
# Description of change
Added assertions for bookmarks tests.
Added 5 of the 8 streams.  Spike card to see about covering the last 3.

# Manual QA steps
None
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tests/test_freshdesk_bookmarks.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+            state_format = self.BOOKMARK_FORMAT
+            calculated_state_formatted = datetime.strftime(calculated_state_as_datetime, state_format)
+
+            stream_to_calculated_state[stream] = calculated_state_formatted
+
+        return stream_to_calculated_state
+
+    # function for verifying the date format
+    def is_expected_date_format(self, date):
```
Comment (kspeer825) [inline tests/test_freshdesk_bookmarks.py:90]: I don't think this method is used anywhere, can be deleted

File: tests/test_freshdesk_bookmarks.py [hunk 2]
```diff
@@ -0,0 +1,218 @@
+"""Test tap check mode and metadata/annotated-schema."""
```
Comment (kspeer825) [inline tests/test_freshdesk_bookmarks.py:1]: This docstring is copy pasta, can be deleted

File: tests/test_freshdesk_bookmarks.py [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+        return_value = {
+            #'start_date':  dt.today() - timedelta(days=5*365),
+            'start_date':  '2016-02-09T00:00:00Z',
+            #'start_date_with_fmt': dt.strftime(start_date, self.START_DATE_FORMAT),
+        }
+
+        return return_value
+
+    @staticmethod
+    def convert_state_to_utc(date_str):
```
Comment (kspeer825) [inline tests/test_freshdesk_bookmarks.py:35]: This is also not used

File: tests/test_freshdesk_bookmarks.py [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+                    self.assertEqual(rec_time[-1], first_bookmark_value)
+
+                    rec_time = []
+                    for record in second_sync_records:
+                        rec_time += record['updated_at'],
+
+                    rec_time.sort()
+                    self.assertEqual(rec_time[-1], second_bookmark_value)
+
+                    # Verify all replication key values in sync 2 are greater than or equal to the simulated bookmark value.
```
Comment (kspeer825) [inline tests/test_freshdesk_bookmarks.py:202]: nice

**Review by kspeer825 — CHANGES_REQUESTED** [review]
Assertions look great, just left some cleanup stuff

File: tests/test_freshdesk_bookmarks.py (hunk 1 above)
Comment (bhtowles) [reply tests/test_freshdesk_bookmarks.py:90]: Deleted

File: tests/test_freshdesk_bookmarks.py (hunk 3 above)
Comment (bhtowles) [reply tests/test_freshdesk_bookmarks.py:35]: Deleted

File: tests/test_freshdesk_bookmarks.py (hunk 4 above)
Comment (bhtowles) [reply tests/test_freshdesk_bookmarks.py:202]: Thanks

File: tests/test_freshdesk_bookmarks.py (hunk 2 above)
Comment (bhtowles) [reply tests/test_freshdesk_bookmarks.py:1]: Deleted

**Review by kspeer825 — APPROVED** [review]

**Comment by bhtowles** [comment]
These changes were merged in pull #46

**Pull request closed without merging**